src/adapters/gumgum.js

```diff
@@ -10,8 +10,16 @@ const GumgumAdapter = function GumgumAdapter() {
 
   const bidEndpoint = `https://g2.gumgum.com/hbid/imp`;
 
-  const WINDOW = global.top;
-  const SCREEN = WINDOW.screen;
+  let WINDOW;
+  let SCREEN;
+
+  try {
+    WINDOW = global.top;
+    SCREEN = WINDOW.screen;
+  } catch (error) {
+    utils.logError(error);
+    return;
```

Comment:
@davidmh 
We'd like to merge this code for now to fix our jsfiddle examples for the 0.16.x release. (see here http://jsfiddle.net/prebid/bhn3xk2j/4/? for example). 

Basically, because JsFiddle sandboxes the execution environment, access to window.top is blocked. Typically on a publisher page this is *not* the case but it causes our examples to stop working. The above code catches this error and will not run GumGum adapter in that case. 

Feel free to work on a solution that would support an sandboxed environment though for the future.